lib/loadTexture.js

```diff
@@ -27,6 +28,7 @@ function loadTexture(texturePath, options) {
     options = defaultValue(options, {});
     options.checkTransparency = defaultValue(options.checkTransparency, false);
     options.decode = defaultValue(options.decode, false);
+    options.keepSource = defaultValue(options.keepSource, false);
```

Comment:
Is keepSource here set to true when loading a Mtl that will require some texture decoding, so that the texture can be duplicated if needed (like in this case where the diffuse and specular are the same JPEG)?

Reply:
Yeah, the texture needs to be decoded for metallic roughness but the source image should stay around when saving the diffuse texture.